kf/kf_unix.go

```diff
@@ -229,12 +232,54 @@ func (b *BPF) LoadTCAttachProgram(ifaceName, direction string) error {
 		if err != nil {
 			return fmt.Errorf("could not get interface %s from id %d: %v", ifaceName, qdisc.Ifindex, err)
 		}
-		if iface.Name == ifaceName && qdisc.Kind == "clsact" {
-			clsactFound = true
+		if iface.Name == ifaceName {
+			switch qdisc.Kind {
+			case "clsact":
+				clsactFound = true
+			case "htb":
+				htbFound = true
+				htbHandle = qdisc.Msg.Handle
+			case "ingress":
+				ingressFound = true
+				ingressHandle = qdisc.Msg.Handle
+			default:
+				log.Info().Msgf("Un-supported qdisc kind for interface %s ", ifaceName)
```

Comment:
Is this going to be too verbose? Or is it useful to a user who is inspecting the logs?

Reply:
I believe it will be useful for debugging on the hypervisors.